components/match_table/commons/match_table.lua

```diff
@@ -414,6 +420,7 @@ function MatchTable:matchFromRecord(record)
 		result = result,
 		game = record.game,
 		date = record.date,
+		bestof = tonumber(record.bestof) or 0,
```

Comment:
why the or 0?

Reply:
That's how MatchGroup/Util does it, I figured when I copy parts of the implementation there i should stick to it

Reply:
just seemed redundant during review
/shrug